indico/modules/events/persons/templates/management/person_list.html

```diff
@@ -0,0 +1,102 @@
+                        <input type="checkbox" value="{{ person.id }}" class="select-row" name="person_id"
+                               {{ 'disabled' if not person.email }}>
+                    </td>
+                    <td class="i-table">{{ person.full_name }}</td>
+                    <td class="i-contributiontable email-column">{{ person.email }}</td>
+                    <td class="i-table affiliation-column">{{ person.affiliation }}</td>
+                    <td class="i-table">
+                        {% if person_data.roles.chairperson %}
+                            <span class="i-label">
+                                {% trans %}Chairperson{% endtrans %}
+                            </span>
+                        {% endif %}
+                    </td>
+                    <td class="i-table">
+                        {% if person_data.contributions %}
+                            <span class="i-label js-count-label"
+                                  data-items="{{ person_data.contributions | tojson | forceescape }}">
+                                {% trans %}Speaker{% endtrans %}
+                                <span class="badge">
+                                    {{ person_data.contributions | length }}
+                                </span>
+                            </span>
+                        {% endif %}
+                    </td>
+                    <td class="i-table">
+                        {% if person_data.subcontributions %}
+                            <span class="i-label js-count-label"
+                                  data-items="{{ person_data.subcontributions | tojson | forceescape }}">
+                                {% trans %}Sub-contrib. Speaker{% endtrans %}
+                                <span class="badge">
+                                    {{ person_data.subcontributions | length }}
+                                </span>
+                            </span>
+                        {% endif %}
+                    </td>
+                    <td class="i-table">
+                        {% if person_data.session_blocks %}
+                            <span class="i-label js-count-label"
+                                  data-items="{{ person_data.session_blocks | tojson | forceescape }}">
+                                {% trans %}Convener{% endtrans %}
+                                <span class="badge">
+                                    {{ person_data.session_blocks | length }}
+                                </span>
+                            </span>
+                        {% endif %}
+                    </td>
+                    <td class="i-table">
+                        {% if person_data.no_account %}
+                            <span class="i-label warning icon-warning"
+                                   title="{% trans %}This user has no Indico account and will not be able to upload materials.{% endtrans %}">
```

Comment:
a bit misleading in case of a pending session manager. maybe mention that he'll get the permissions once he registers with the same email address?